### PR TITLE
integrations: Add notes for ReviewBot

### PR DESCRIPTION
**Pull request by andrewallen00**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@andrewallen00, it looks like you've already signed the Dropbox CLA. Thanks!

File: webhook-integrations/reviewbot/notes.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# ReviewBot Integration
+The ReviewBot integration is designed to keep track of reviews on an iOS
+app, an Android app, a Yelp website, or a iTunes podcast. It notifies you on
+Slack or through several other services. You can set parameters for which
+reviews you receive notifications, for example to only get a notification
+for a 5 star review. This is the official website for the integration:
```
Comment (YagoGG) [inline webhook-integrations/reviewbot/notes.md]: It think *to only get notifications with 5-stars reviews* would be better.

File: webhook-integrations/reviewbot/notes.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Problems
+
+There was no available documentation. As a result, I could not
+figure out how to add a webhook URL. My guess is that ReveiwBot was built
+to support specific services (such as Slack or Twitter), without a way to
+implement a webhook. I contacted the ReviewBot team, to see if they could
+provide any additional information. They responded and said that it was not
+currently possible to add a webook, but they could add a general webhook
+option, which would make a ReveiwBot integration possible. Unfortunately,
```
Comment (YagoGG) [inline webhook-integrations/reviewbot/notes.md]: There's a typo here. "ReviewBot" instead of "ReveiwBot".

**Comment by andrewallen00** [comment]
@showell thanks!

**Pull request merged**